app/controllers/bgeigie_imports_controller.rb

```diff
@@ -34,6 +34,13 @@ def approve
     redirect_to @bgeigie_import
   end
 
+  def fixdrive
+    @bgeigie_import = BgeigieImport.find(params[:id])
```

Comment:
#approve is also unscoped. I also noticed we should have this action in the require_moderator list as well. Will push that change in a second.